apps/src/music/player/MusicLibrary.ts

```diff
@@ -2,6 +2,16 @@ import {ResponseValidator} from '@cdo/apps/util/HttpClient';
 import {Key} from '../utils/Notes';
 
 export default class MusicLibrary {
+  private static instance: MusicLibrary;
```

Comment:
Separate but somewhat related change: made the current library accessible via singleton rather than having to pass it to all the components and have `isInitialized` checks everywhere